build.config.ts

```diff
@@ -0,0 +1,19 @@
+import { fileURLToPath } from "node:url";
+import { defineBuildConfig } from "unbuild";
+
+export default defineBuildConfig({
+  entries: [
+    "src/index.ts",
+    {
```

Comment:
Actually, using `mkdist` might be overcomplicating things. Could you please try just referencing all source files like `services/resend` in the `entries` option?

Like this:
```ts
export default defineBuildConfig({
  entries: [
    "src/index",
    "services/resend",
    "services/plunk"
    // etc.
  ],
  // You should now be able to omit the `externals`!
}
```

Rollup should automatically resolve these.